packages/core/solidity/src/erc721.ts

```diff
@@ -216,25 +216,34 @@ const functions = defineFunctions({
   _update: {
     kind: 'internal' as const,
     args: [
+      { name: 'from', type: 'address' },
       { name: 'to', type: 'address' },
       { name: 'tokenId', type: 'uint256' },
       { name: 'auth', type: 'address' },
     ],
-    returns: ['address'],
+    comments: [
+      '/// @dev Hook that is called during any token transfer. This includes minting and burning. Override this function to add custom logic for token transfers.',
+    ],
   },
 
   tokenURI: {
     kind: 'public' as const,
     args: [{ name: 'tokenId', type: 'uint256' }],
     returns: ['string memory'],
-    mutability: 'view' as const,
+    mutability: 'view',
```

Comment:
Why was `as const` removed?
```suggestion
    mutability: 'view' as const,
```